website/README.md

```diff
@@ -1,33 +1,33 @@
-# Website
+# Chaos Mesh Website
 
 This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
 
 ### Installation
 
-```
-$ yarn
+```sh
+yarn
 ```
 
 ### Local Development
 
-```
-$ yarn start
+```sh
+yarn start
 ```
 
-This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+This command starts a local development server and open up a browser window. Most changes are reflected live without having to restart the server.
```

Comment:
It seems that "opens" is more reasonable?

Reply:
Yep. It should be `opens`.